src/test/java/org/openmrs/addonindex/scheduled/FetchDetailsToIndexTest.java

```diff
@@ -49,20 +53,32 @@ public void testParsingConfigXmlForRequiredModuleVersion() throws Exception {
 				hasProperty("version", is("?"))
 		)));
 	}
-	
-	@Test
+
+    @Test
+	public void testParsingConfigXmlForSettingModulePackageAndId() throws Exception {
+	    FetchDetailsToIndex task = new FetchDetailsToIndex(null, null);
+	    AddOnVersion version = new AddOnVersion();
+	    AddOnInfoAndVersions addOnInfoAndVersions = new AddOnInfoAndVersions();
+	    task.handleConfigXml(getFileAsString("config.withRelativePathDtd.xml"), version, addOnInfoAndVersions, false);
+	    assertThat(addOnInfoAndVersions.getModulePackage(), is("org.openmrs.module.mdrtb"));
+	    assertThat(addOnInfoAndVersions.getModuleId(), is("mdrtb"));
+	}
```

Comment:
This test is good.

So you know, for completeness you would also want to have a test where:

* there are two versions
* the moduleId or modulePackage changes between them
* the test ensures that the id/package are taken from the latest version, not the earliest version

The philosophy of test-driven development, or behavior-driven development, says that if this behavior is in the code, you should have started by writing a test for it. :-)

(I'm not going to hold up merging the PR for this, but you're welcome to add it in another commit.)